lib/std/rand.zig

```diff
@@ -31,11 +31,9 @@ const ziggurat = @import("rand/ziggurat.zig");
 const maxInt = std.math.maxInt;
 
 /// Fast unbiased random numbers.
+/// Note: this is not a cryptographically secure random number generator.
 pub const DefaultPrng = Xoroshiro128;
 
-/// Cryptographically secure random numbers.
-pub const DefaultCsprng = Gimli;
```

Comment:
I would keep this here and switch it over when the new one arrives.

Reply:
The interface may be different, though.